charmcraft/commands/store/__init__.py

```diff
@@ -636,6 +636,31 @@ def _get_lib_info(*, full_name=None, lib_path=None):
         full_name=full_name, path=lib_path, lib_name=lib_name, charm_name=charm_name)
 
 
+def _get_libs_from_tree(charm_name=None):
+    """Get library info from the directories tree (for a specific charm if specified).
+
+    It only pays attention to the proper disk structures, if there.
+    """
+    local_libs_data = []
+
+    if charm_name is None:
+        base_dir = pathlib.Path('lib') / 'charms'
+        charm_dirs = sorted(base_dir.iterdir()) if base_dir.exists() else []
+    else:
+        base_dir = pathlib.Path('lib') / 'charms' / charm_name
+        charm_dirs = [base_dir] if base_dir.exists else []
```

Comment:
exists should be exists() but similarly
```suggestion
        charm_dirs = [base_dir] if base_dir.is_dir() else []
```

Reply:
Thanks!